projects/avodata/firstStage.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import math
+
+def factorial(n):
+	return int(math.gamma(n + 1))
+
+
+def nChoosek(n, k):
```

Comment:
We use the Google Style Guide:
https://github.com/google/styleguide/blob/gh-pages/pyguide.md

Please use snake_case for method names: `nChoosek->n_choose_k`

Reply:
done